examples/README.md

```diff
@@ -202,3 +202,70 @@ In case of the specific use-case where Manufacturing Server is located in differ
     - import_role:
         name: copy_ownership_vouchers_from_localhost
 ```
+
+## Use Custom FDO Keys and Certificates
+
+In cases where manufacturing and owner servers belong to different entities,
+it is possible to use custom keys and certificates instead of having them
+generated by the manufacturing server. The following examples show how to
+distribute keys and certificates to FDO servers. The collection also includes
+a role for generating all or a subset of keys and certificates if needed.
+
+```yaml
+---
+- name: Generate FDO Keys and Certificates
+  hosts: localhost
+  become: true
```

Comment:
Since the default destination is `/var/tmp/fdo` why do we need privilege escalation?

Reply:
@fabiendupont for installing `fdo-admin-cli` if it wasn't installed. All these roles might or might not require `become: true` depending on the Ansible user, it's just an example.

Reply:
Regarding out-of-band generation, you'll need to look at the role. The examples cover only basic scenarios and I didn't want to change that for now. It's also no different than before for rendezvous and owner, what's changed is the manufacturer, but I kept the same pattern. In any case, there's a lot of documenting to do later.

Reply:
Actually, it could be a good idea to at least point to the relevant section in the doc. Will do that.